### PR TITLE
Post about Sunny tech

### PR DESCRIPTION
**Pull request by VincentBrule**

### CODE REVIEW

File: posts/2022-07-02-sunny-tech-en.adoc [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+. Methodologies
+. Backend
+. Data
+. Web & Mobile & UI
+. Architecture
+. Atypical
+
+The organizers have prepared a _great schedule for us_.
+Thanks to the diversity of the topics presented, there was really something for every taste and profile.
+If you are interested, https://sunny-tech.io/schedule/[the program is available on the site].
```
Comment (thinkmorestupidless) [inline posts/2022-07-02-sunny-tech-en.adoc:24]: Feels a little like switching to present tense here - perhaps...

``suggestion
If you are interested, https://sunny-tech.io/schedule/[the program is still available on the site].
```

File: posts/2022-07-02-sunny-tech-en.adoc [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+The presentations had different formats.
+First, the classic 50-minute presentations.
+Then, workshops to learn by doing of varying length.
+And finally, short talk of 20 minutes.
```
Comment (thinkmorestupidless) [inline posts/2022-07-02-sunny-tech-en.adoc:28]:
```suggestion
And finally, short talks of 20 minutes.
```

File: posts/2022-07-02-sunny-tech-en.adoc [hunk 3]
```diff
@@ -0,0 +1,227 @@
...
+
+=== The organization
+Before presenting you my favorite discoveries of *Sunny Tech*, I would like to thank all the volunteers who made this event possible after several years of absence due to Covid.
```
Comment (thinkmorestupidless) [inline posts/2022-07-02-sunny-tech-en.adoc:31]: You could go with

```suggestion
Before presenting to you my favourite discoveries of *Sunny Tech*, I would like to thank all the volunteers who made this event possible after several years of absence due to Covid.
```

or 

```suggestion
Before talking about my favourite discoveries of *Sunny Tech*, I would like to thank all the volunteers who made this event possible after several years of absence due to Covid.
```

File: posts/2022-07-02-sunny-tech-en.adoc [hunk 4]
```diff
@@ -0,0 +1,227 @@
...
+I would also like to highlight all the actions for the environment that have been implemented, such as:
+
+. the possibility of not taking goodies and in exchange give the amount of these objects to an https://twitter.com/SunnyTech_MTP/status/1541701793322831873[association]
+. a special booth for a local association, this year we were able to meet https://labelbleu.assoconnect.com/collect/description/233850-v-bulletin-d-adhesion-a-l-association-labelbleu-2022[LabelBleu]
+. a 0-waste caterer who uses only reusable containers such as glass jars (and the food was very good, thanks to https://instant-bocal.com/[Instant bocal])
+
+.Yum !
+image::food.jpeg[Glass jars from Sunny Tech's caterer, width = 800]
+
+I will now present you my favorites of this conference!
```
Comment (thinkmorestupidless) [inline posts/2022-07-02-sunny-tech-en.adoc:46]: see above

File: posts/2022-07-02-sunny-tech-en.adoc [hunk 5]
```diff
@@ -0,0 +1,227 @@
...
+Indeed, the faces coming from this group of photos have a particularity: _the eyes are always placed in the same place_.
+Therefore, the AI will try to create images as close as possible to the real ones and will therefore also place the eyes in the same place.
+The most naive solution is to look at the position of the eyes on the image and if it matches the known position, the image is false.
+But this solution is not very robust to changes in the pictures, so the presenter developed a more advanced method.
+
+To do this, it has set up a service that retrieves and stores images from the site_thispersondoesnotexist_.
+It has made several optimizations in order not to store the raw images on the server, as this would take up too much disk space.
+Then, when you want to detect a fake face, the tool will compare with its database and according to a score, send back yes or no.
+If you want to test, Mathis hosts a https://fada.h25.io/[website] allowing to detect fake images directly from the browser.
+Moreover, this web site displays interesting information such as look-alikes, when the fake image was generated, or the eyes technique explained before.
```
Comment (thinkmorestupidless) [inline posts/2022-07-02-sunny-tech-en.adoc:125]:
```suggestion
Moreover, this website displays interesting information such as look-alikes, when the fake image was generated, or the eyes technique explained before.
```

File: posts/2022-07-02-sunny-tech-en.adoc [hunk 6]
```diff
@@ -0,0 +1,227 @@
...
+
+This talk is really well popularized and we are directly captured by the subject thanks to a very good speaker!
```
Comment (thinkmorestupidless) [inline posts/2022-07-02-sunny-tech-en.adoc:127]: i'm not 100% clear what you're trying to say here - do you mean the talk was about a popular subject? Or that it was clearly explained? Sorry, i'm not quite getting this :(

File: posts/2022-07-02-sunny-tech-en.adoc [hunk 7]
```diff
@@ -0,0 +1,227 @@
...
+
+=== "Il était une fois... les navigateurs" (Once upon a time... the browsers)
+
+*Noël Macé* and *Pierre Tibulle* presented us an interesting subject with a very original presentation!
+Indeed, while Noël explained the origins of browsers and their evolution through time, Pierre drew the slides in _real time_ on the stage.
+
+.Once upon a time... the browsers
+image::live-drawing.jpg[Once upon a time... the browsers, width = 800]
+
+Thanks to this duet, we can get hooked directly to the story that is told to us.
```
Comment (thinkmorestupidless) [inline posts/2022-07-02-sunny-tech-en.adoc:137]: could be

```suggestion
Thanks to this duet, we can get hooked directly into the story that is told to us.
```

or 

```suggestion
Thanks to this duet, we can get directly hooked into the story that is told to us.
```

probably the former is better

File: posts/2022-07-02-sunny-tech-en.adoc [hunk 8]
```diff
@@ -0,0 +1,227 @@
...
+I admire a lot the people who manage to go out of the frame to present their subject, a big bravo to them!
```
Comment (thinkmorestupidless) [inline posts/2022-07-02-sunny-tech-en.adoc:138]:
```suggestion
I really admire the people who manage to go out of the frame to present their subject, a big bravo to them!
```

File: posts/2022-07-02-sunny-tech-en.adoc [hunk 9]
```diff
@@ -0,0 +1,227 @@
...
+If you're a Mario Kart player on Nintendo Switch, I highly recommend watching this talk a few times!
+
+=== "Camping Paradigme : Organise tes vacances en mode Agile !" (Camping Paradigm: Organize your vacations in Agile mode !)
+
+https://twitter.com/Laurie78645718[Laurie Coquillat] imagined and created a board game to explain *Agile* methods in a playful way.
+
+.Camping Paradigm: Organize your vacations in Agile mode !
+image::agile1.jpg[Camping Paradigm: Organize your vacations in Agile mode !, width = 800]
+
+In this workshop, you have to organize your group vacation with unforeseen events and constraints in the best possible way.
```
Comment (thinkmorestupidless) [inline posts/2022-07-02-sunny-tech-en.adoc:200]:
```suggestion
In this workshop, you had to organize your group vacation with unforeseen events and constraints in the best possible way.
```

File: posts/2022-07-02-sunny-tech-en.adoc [hunk 10]
```diff
@@ -0,0 +1,227 @@
...
+This parallel with an IT project allows participants to discover the world of agility in an original way.
+
+.The game board
+image::agile2.jpg[The game board, width = 800]
+
+There are several points to take into account during our adventure such as the mode of transport used, the activities, the meals or the chosen accommodation.
+During the game, you may come across some unexpected situations that you have to deal with as a group.
+
+Laurie is present throughout the game to guide and help us.
```
Comment (thinkmorestupidless) [inline posts/2022-07-02-sunny-tech-en.adoc:209]:
```suggestion
Laurie was present throughout the game to guide and help us.
```

File: posts/2022-07-02-sunny-tech-en.adoc [hunk 11]
```diff
@@ -0,0 +1,227 @@
...
+At the end of the game, she highlights the parallels with our IT projects to explain the different Agile frameworks and their use cases.
```
Comment (thinkmorestupidless) [inline posts/2022-07-02-sunny-tech-en.adoc:210]:
```suggestion
At the end of the game, she highlighted the parallels with our IT projects to explain the different Agile frameworks and their use cases.
```

File: posts/2022-07-02-sunny-tech-fr.adoc [hunk 12]
```diff
@@ -0,0 +1,225 @@
...
+Les organisateurs nous ont concocté un _planning aux petits oignons_.
+Grâce à la diversité des sujets présentés, il y en avait vraiment pour tous les goûts et tous les profils.
+Si vous êtes intéressé, https://sunny-tech.io/schedule/[le programme est disponible sur le site].
+Les présentations étaient sous différentes formes.
+Tout d'abord, les présentations classiques d'une durée de 50 minutes.
+Ensuite, des ateliers pour apprendre en pratiquant d'une durée variable.
+Et pour finir, les quickies de 20 minutes.
+
+=== L'organisation
+Avant de vous présenter mes découvertes coup de coeur de *Sunny Tech*, j'aimerais remercier tous les bénévoles qui ont rendu cet évènement possible après plusieurs années d'absence dues à la Covid.
```
Comment (LaurieCoq) [inline posts/2022-07-02-sunny-tech-fr.adoc:31]: événement

File: posts/2022-07-02-sunny-tech-fr.adoc [hunk 13]
```diff
@@ -0,0 +1,225 @@
...
+[quote, Définition Wikipédia d'un dark pattern]
+Un dark pattern ou dark UX (en français : interface truquée) est une interface utilisateur qui a été spécialement conçue pour tromper ou manipuler un utilisateur.
+
+Concernant l'impact écologique de tous nos appareils, plusieurs solutions sont possibles aussi :
+
+1. Louer au lieu d'acheter, sur https://commown.coop/[Commown] par exemple
+2. Adopter un opérateur responsable comme https://telecoop.fr/[Telecoop]
+3. Utiliser un système d'exploitation respectueux comme https://e.foundation/[e]
+4. Garder ses appareils le plus longtemps possible au lieu de succomber en permanence au dernier produit à la mode
+5. Favoriser des appareils facilement réparables, https://www.indicereparabilite.fr/[l'indice de réparabilité] peut vous aider à y voir plus clair. Le gouvernement français a crée une plateforme pour cela aussi : https://longuevieauxobjets.gouv.fr/
```
Comment (LaurieCoq) [inline posts/2022-07-02-sunny-tech-fr.adoc:81]: a créé

File: posts/2022-07-02-sunny-tech-fr.adoc [hunk 14]
```diff
@@ -0,0 +1,225 @@
...
+=== Comment j'ai développé le détecteur de deepfakes le plus puissant du monde pour 100€
+
+https://twitter.com/mathishammel[Mathis Hammel] nous a présenté sa solution maison pour détecter les *deepfakes*.
+Le site https://thispersondoesnotexist.com[thispersondoesnotexist] génère de faux visages gratuitement et en libre accès.
+
+.Faux visage récupéré sur thispersondoesnotexist
+image::fake-face.jpeg[Faux visage récupéré sur thispersondoesnotexist, width = 800]
+
+Des individus mal intentionnés se servent de ce genre de services pour créer des faux profils en masse sur les réseaux sociaux.
+Pour remédier à cela, Mathis a crée de toute pièce une solution pour les détecter.
```
Comment (LaurieCoq) [inline posts/2022-07-02-sunny-tech-fr.adoc:96]: a créé

File: posts/2022-07-02-sunny-tech-fr.adoc [hunk 15]
```diff
@@ -0,0 +1,225 @@
...
+.Résumé de la présentation réalisé par @ptibulle
+image::deep-fake.jpeg[Résumé de la présentation réalisé par @ptibulle, width = 800]
+
+Le speaker a commencé sa réflexion avec une solution simple mais efficace.
+Avant de l'expliquer, nous devons revenir sur les méthodes utilisées pour réaliser cette IA.
+
+La technique d'intelligence artificielle utilisée pour générer les images est un https://fr.wikipedia.org/wiki/R%C3%A9seaux_antagonistes_g%C3%A9n%C3%A9ratifs[GAN] (réseau antagoniste génératif).
+Ce type de modèle se base sur trois parties :
+
+1. un *generator* qui va produire des images aléatoires au début puis de plus en plus réalises au fil de l'entraînement
```
Comment (LaurieCoq) [inline posts/2022-07-02-sunny-tech-fr.adoc:107]: réalistes

File: posts/2022-07-02-sunny-tech-fr.adoc [hunk 16]
```diff
@@ -0,0 +1,225 @@
...
+
+=== Il était une fois... les navigateurs
+
+*Noël Macé* et *Pierre Tibulle* nous ont présenté un sujet intéressant avec une présentation très originale !
+En effet, pendant que Noël explique les origines des navigateurs et leurs évolutions à travers le temps, Pierre dessine les slides en _temps réel_ sur la scène.
+
+.Il était une fois... les navigateurs
+image::live-drawing.jpg[Il était une fois... les navigateurs, width = 800]
+
+Grâce à ce duo, on accroche directement à l'histoire qui nous ait raconté.
```
Comment (LaurieCoq) [inline posts/2022-07-02-sunny-tech-fr.adoc:136]: qui nous est racontée.

File: posts/2022-07-02-sunny-tech-fr.adoc [hunk 17]
```diff
@@ -0,0 +1,225 @@
...
+. _des personnes tech qui parlent de technique_. Autrement dit, un DevRel a pratiqué, codé et testé une solution avant de la présenter
+. _être sincère_, c'est-à-dire, ne pas donner un discours commercial sans fond seulement dans l'objectif de vendre mais un discours sincère grâce à leurs expériences
+
+Je vous invite vivement à regarder la rediffusion de ce moment pour en savoir plus !
+https://noti.st/aurelievache/PJZgaj[Les slides sont aussi disponibles].
+
+=== Éclater ses collègues à Mario Kart
+
+Ce talk décalé présenté par https://twitter.com/_yannbertrand[Yann Bertrand] nous explique une multitude de techniques pour ne plus finir dernier à *Mario Kart* !
+Je trouve ça génial que ce fan de jeux vidéos et de compétitions e-sports puissent venir nous parler de sa passion dans une conférence comme Sunny Tech.
```
Comment (LaurieCoq) [inline posts/2022-07-02-sunny-tech-fr.adoc:178]: leur passion

File: posts/2022-07-02-sunny-tech-fr.adoc [hunk 18]
```diff
@@ -0,0 +1,225 @@
...
+
+Pour les joueurs de Mario Kart Deluxe 8, voici les principales techniques que je ne connaissais pas :
+
+. Si vous avez une pièce en objet principal, *gardez-la* car le prochain objet sera forcément un *objet de protection*
+. Quand une carapace rouge vous arrive dessus, vous pouvez rentrer dans un mur au dernier moment pour l'éviter
+. Pour *éviter une carapace bleue*, vous pouvez utiliser un *champignon* au moment où la carapace vous tombe dessus
+
+Par la suite, Yann nous a présenté plusieurs méthodes avancées, mais pour être honnête, je ne pense pas être capable de les reproduire un jour...
+
+Le week-end après la conférence a été l'occasion de tester toutes ses techniques et à ma plus grande surprise, je me suis vraiment amélioré !
```
Comment (LaurieCoq) [inline posts/2022-07-02-sunny-tech-fr.adoc:188]: toutes ces techniques

File: posts/2022-07-02-sunny-tech-fr.adoc [hunk 19]
```diff
@@ -0,0 +1,225 @@
...
+
+=== Camping Paradigme : Organise tes vacances en mode Agile !
+
+https://twitter.com/Laurie78645718[Laurie Coquillat] a imaginé et crée un jeu de société pour expliquer les méthodes *Agile* de façon ludique.
+
+.Camping Paradigme : Organise tes vacances en mode Agile !
+image::agile1.jpg[Camping Paradigme : Organise tes vacances en mode Agile !, width = 800]
+
+Dans cet atelier, vous devez organiser vos vacances en groupe avec des imprévus et des contraintes de la meilleure façon possible.
+Ce parallèle avec un projet IT permets aux participants de découvrir le monde de l'agilité d'une façon originale.
```
Comment (LaurieCoq) [inline posts/2022-07-02-sunny-tech-fr.adoc]: permet aux participants

File: posts/2022-07-02-sunny-tech-fr.adoc [hunk 20]
```diff
@@ -0,0 +1,225 @@
...
+
+.Plateau du jeu
+image::agile2.jpg[Plateau du jeu, width = 800]
+
+Il y a plusieurs points à prendre en compte durant notre aventure comme le mode de transport utilisé, les activités, les repas ou encore le logement choisi.
+Au cours de la partie, on peut tomber sur des cases galères représentant des imprévus auxquels il faut faire face en groupe.
+
+Laurie est présente tout au long de la partie pour nous guider et nous aider.
+À la fin de la partie, elle mets en valeurs les parallèles avec nos projets habituels pour nous expliquer les différents frameworks Agile et leurs cas d'utilisation.
```
Comment (LaurieCoq) [inline posts/2022-07-02-sunny-tech-fr.adoc]: elle met

File: posts/2022-07-02-sunny-tech-fr.adoc [hunk 21]
```diff
@@ -0,0 +1,225 @@
...
+
+Si jamais vous allez dans une conférence où Laurie fait cet atelier, allez-y les yeux fermés.
+Même si vous travaillez au quotidien avec l'esprit Agile, ce workshop permet de redécouvrir les bases et l’intérêt de ces méthodes de façon ludique !
+
+== Merci Sunny Tech
+C'était ma première fois à Sunny Tech et sûrement pas la dernière !
+
+Les billets pour la conférence ont un prix vraiment raisonnable (90 euros pour les deux jours si vous vous y prenez à la dernière minute) avec une organisation soignée et des bénévoles au top !
```
Comment (LaurieCoq) [inline posts/2022-07-02-sunny-tech-fr.adoc:217]: si vous ne vous y prenez pas à la dernière minute

File: posts/2022-07-02-sunny-tech-fr.adoc [hunk 22]
```diff
@@ -0,0 +1,225 @@
...
+N'hésitez pas à proposer des sujets pour l'année prochaine ou encore à participer à l'organisation car ils ont besoin de bénévoles pour rendre cet évènement possible.
```
Comment (LaurieCoq) [inline posts/2022-07-02-sunny-tech-fr.adoc:218]: événement

File: posts/2022-07-02-sunny-tech-fr.adoc [hunk 23]
```diff
@@ -0,0 +1,225 @@
...
+
+Merci encore à tout le monde et à l'année prochaine sous le soleil de Montpellier !
+
+.Merci Sunny Tech par https://www.instagram.com/mel.owski/[Melowski]
+image::flamant.png[Merci Sunny Tech, width = 800]
+
+_J'aimerai aussi remercier https://www.instagram.com/mel.owski/[Melowski] qui a réalisé les deux illustrations avec les flamants rose spécialement pour cet article !_
```
Comment (LaurieCoq) [inline posts/2022-07-02-sunny-tech-fr.adoc:225]: J'aimerais

File: posts/2022-07-02-sunny-tech-en.adoc (hunk 3 above)
Comment (VincentBrule) [reply posts/2022-07-02-sunny-tech-en.adoc:31]: I like the second one thanks 👍

File: posts/2022-07-02-sunny-tech-fr.adoc (hunk 21 above)
Comment (VincentBrule) [reply posts/2022-07-02-sunny-tech-fr.adoc:217]: 90 euros c'est le prix fort, pas early bird c'est pour ça que j'ai mis si on s'y prend au dernier moment :)

File: posts/2022-07-02-sunny-tech-en.adoc [hunk 24]
```diff
@@ -0,0 +1,229 @@
...
+
+=== "Il était une fois... les navigateurs" (Once upon a time... the browsers)
+
+*Noël Macé* and *Pierre Tibulle* presented us an interesting subject with a very original presentation!
+Indeed, while Noël explained the origins of browsers and their evolution through time, Pierre drew the slides in _real time_ on the stage.
+
+.Once upon a time... the browsers
+image::live-drawing.jpg[Once upon a time... the browsers, width = 800]
+
+Thanks to this duet, we can get hooked directly into the story that is told to us.
```
Comment (ckipp01) [inline posts/2022-07-02-sunny-tech-en.adoc:139]:
```suggestion
Thanks to this duet, we can get hooked directly into the story that is told to us.
```
I'm not 100% sure what this means. Does this mean there were engaging speakers?

File: posts/2022-07-02-sunny-tech-en.adoc [hunk 25]
```diff
@@ -0,0 +1,229 @@
...
+
+=== "Mais, à quoi ça sert un DevRel ?" (But, what is a DevRel for?)
+
+We had the privilege to receive two unicorns on stage to explain us the job of **DevRel** (Developer Relations).
+
+.But, what is a DevRel for?
+image::devrel.jpg[But, what is a DevRel for?, width = 800]
+
+I really enjoyed this talk because the term *DevRel* was obscure to me.
+I imagined a person who goes from conference to conference to sell his company.
```
Comment (ckipp01) [inline posts/2022-07-02-sunny-tech-en.adoc:152]:
```suggestion
I imagined a person who goes from conference to conference to sell their company.
```
Better to be gender agnostic here.

**Review by ckipp01 — APPROVED** [review]
Cool post @VincentBrule! Sounds like an interesting conference! I left some grammar suggestions.

File: posts/2022-07-02-sunny-tech-en.adoc [hunk 26]
```diff
@@ -0,0 +1,229 @@
...
+The goal of the generator is to produce the most realistic image possible to deceive the discriminator and the goal of the discriminator is to detect as well as possible the false images generated.
+After many training cycles, the two modules will improve and become more and more efficient.
+
+The first trick to detect false images generated by this model comes from the *image bank used*.
+Indeed, the faces coming from this group of photos have a particularity: _the eyes are always placed in the same place_.
+Therefore, the AI will try to create images as close as possible to the real ones and will therefore also place the eyes in the same place.
+The most naive solution is to look at the position of the eyes on the image and if it matches the known position, the image is false.
+But this solution is not very robust to changes in the pictures, so the presenter developed a more advanced method.
+
+To do this, it has set up a service that retrieves and stores images from the site_thispersondoesnotexist_.
```
[the remark this answers is not in the document]
Comment (VincentBrule) [reply posts/2022-07-02-sunny-tech-en.adoc:125]: Nice catch :D

**Pull request merged**